mgmtsystem/models/mgmtsystem_system.py

```diff
@@ -0,0 +1,37 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2012 Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields
+
+
+def own_company(self):
+    return self.env.user.company_id.id
+
+
+class mgmtsystem_system(models.Model):
+
+    _name = 'mgmtsystem.system'
+    _description = 'System'
+
+    name = fields.Char(string='System', required=True)
+    manual = fields.Many2one('document.page', string='Manual')
```

Comment:
@dreispt what do you want to do with this field? I suggest to move it to the mgmtsystem_manuals module